src/pages-and-resources/discussions/app-config-form/apps/shared/ExternalDocumentations.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import {
+  Hyperlink, MailtoLink,
+} from '@edx/paragon';
+
+import messages from '../lti/messages';
+
+function ExternalDocumentations({
+  documentationUrls,
+  intl,
+  title,
+}) {
+  const appInstructions = Object.keys(documentationUrls);
+  return (
+    <div className="pt-4">
+      <h4>{intl.formatMessage(messages.linkTextHeading)}</h4>
+      <div className="small text-muted">
+        {appInstructions.map((instructionType) => (
+          documentationUrls[instructionType]
+          && instructionType !== 'contactEmail'
+          && (
+            <div key={instructionType}>
+              <FormattedMessage
+                {...messages.linkText}
+                values={{
+                  link: (
+                    <Hyperlink
+                      destination={documentationUrls[instructionType]}
```

Comment:
you'd need to add  `target="_blank" rel="noopener noreferrer"` as well

Reply:
yes removed that because the icon start appearing with a link when we add `target="_blank"` property to the hyperlink